src/modules/pass.js

```diff
@@ -630,7 +641,8 @@ PassFF.Pass = (function () {
         .map(i => getItemQuality(i, urlStr))
         .filter(i => (i.quality >= 0))
         .sort((i1,i2) => (i2.quality - i1.quality))
-        .map(i => i.item);
+        .map(i => i.item)
+        .filter(i => !i.isFiltered);
```

Comment:
I think the state `isFiltered` quite misleading: we have to think about it: "Okay, it is filtered, so it is displayed, isn't it? Oops"
I would prefer `isVisible`.
* isFiltered -> isVisible
* isInFilter -> isVisible

On the other hand, I would keep the parameter "filterFieldRegex" because this could be a blacklist or a whitelist.

Reply:
I just realized the entries can also be filtered by the context : see the "Display contextual" button in the toolbar menu.

`isHidden` could be an alternative.

Reply:
I like it. Will show up in my next push.